app/services/payments/process_credit_purchase.rb

```diff
@@ -25,8 +25,14 @@ def initialize(user, credits_count, purchase_options:)
     end
 
     def call
-      process_purchase
-      create_credits if success?
+      # we expect at least one of :stripe_token, :organization_id, or :selected_card to process
+      if purchase_options.compact_blank.present?
+        process_purchase
+        create_credits if success?
+      else
+        self.error = "Please select a payment method"
```

Comment:
this could reasonably be internationalized (the stripe gem doesn't have localized error messages but that doesn't mean we shouldn't).

Reply:
![image](https://user-images.githubusercontent.com/47985/144382092-036cf40f-b600-42a1-bec1-551fe79f0885.png)